modules/api/README.md

```diff
@@ -0,0 +1,50 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
+for interacting with the core LORIS functionality without requiring backend
+SQL access.
+
+## Intended Users
+
+The module has 2 intended use cases:
+1. Users (or creators) of LORIS data that are external to the LORIS core
+   software that must extract or import data from LORIS.
+2. Internal LORIS modules which retrieve data from the API.
+
+## Scope
+
+The API only provides information about concepts that are cross-module
+or data that is of general concern to users. 
+
+NOT in scope:
+
+- Data and functionality that is specific to a single module, and not of general
+  nature.
+- A graphical frontend for interacting with the API.
+
+## Permissions
+
+The `api` module is a public module so that the `login` endpoint can be accessed,
+but otherwise all requests require a token (retrieved from the `login` endpoint)
+or the user accessing the API to be logged in to LORIS.
```

Comment:
```suggestion
for the user accessing the API to be logged in to LORIS.
```

Reply:
Could add that the user need to be logged in via `the web interface` as the concept of being `logged in` can be vague.

Reply:
"or" was correct. You either need a login token from the API without being logged in *or* you need to be logged in to loris as a regular user.